pkgs/by-name/he/heliocron/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "heliocron";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "mfreeborn";
+    repo = "heliocron";
+    rev = "v${version}";
+    hash = "sha256-5rzFz29Bpy2GR6bEt2DdCq9RtpdcY3SK/KnZrBrHUvk=";
+  };
+
+  cargoHash = "sha256-RjYE2TBvHnBgiwJi/Ys+wAmJoMHJuimcD07FOqwkcis=";
+
+  meta = with lib; {
```

Comment:
`with lib;` is also considered harmful https://github.com/NixOS/nixpkgs/issues/208242

Reply:
Makes perfect sense, I mostly copied the `package.nix` contents from a different package (that I later found it was apparently in the "deprecated" `pkgs/applications/` directory) so I assumed it was standard.